src/libraries/System.IO.Pipes/tests/PipeStreamConformanceTests.cs

```diff
@@ -502,8 +502,6 @@ public async Task ReadAsync_DisconnectDuringRead_Returns0()
         }
 
         [PlatformSpecific(TestPlatforms.Windows)] // Unix named pipes are on sockets, where small writes with an empty buffer will succeed immediately
-        [SkipOnPlatform(TestPlatforms.LinuxBionic, "SElinux blocks UNIX sockets in our CI environment")]
```

Comment:
this was just a copy paste error (the test is marked as `[PlatformSpecific(TestPlatforms.Windows)] ` so we don't need to skip it on other platforms)